### DISCUSSION TITLE
Logo for the project

### DISCUSSION THREAD

**Discussion by jcubic** [halide/Halide#6841] [Show and tell]
I've read about your project on [Reddit](https://www.reddit.com/r/ProgrammingLanguages/comments/v0kjbr/how_has_learning_about_pl_design_helped_you_in/). Do you maybe need a logo for you project? I can see if I can design something that will look good. But I don't promise that I will come up with something.

I've created a logo for a few languages already:

Kawa Scheme:
![](https://upload.wikimedia.org/wikipedia/commons/c/ce/Kawa-logo.svg)
LIPS Scheme (my own project)
![](https://raw.githubusercontent.com/jcubic/lips/master/assets/lips.svg)

You can see more of my logos [here](https://jakub.jankiewicz.org/logo.php).

**Comment by mcourteaux** [comment]
I need a Halide T-shirt... :laughing:

**Comment by mcourteaux** [comment]
![image](https://github.com/user-attachments/assets/a27f8be9-48d4-4a5b-b826-1ddf228a5503)

ChatGPT is being helpful. Do we order a batch? 🥴 😋

**Reply by steven-johnson** [reply]
> ChatGPT is being helpful

I'd rather not use any AI-generated art; I have no idea whose actual art is being plagiarized... >:-(

**Reply by abadams** [reply]
There are some models that are only trained on images that have been licensed for the purpose. Firefly is the one I'm aware of.
